compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -84,6 +83,37 @@ impl<'a> Interpreter<'a> {
         Ok(result)
     }
 
+    fn call_builtin(
+        &mut self,
+        function: FuncId,
+        arguments: Vec<(Value, Location)>,
+        location: Location,
+    ) -> IResult<Value> {
+        let attributes = self.interner.function_attributes(&function);
+        let func_attrs = attributes.function.as_ref()
+            .expect("all low level functions must contain a function  attribute which contains the opcode which it links to");
+
+        let name = self.interner.function_name(&function);
+
+        if let Some(builtin) = func_attrs.builtin() {
+            let item = format!("Evaluation for builtin functions like {name}");
+            Err(InterpreterError::Unimplemented { item, location })
+        } else if let Some(lowlevel) = func_attrs.foreign() {
```

Comment:
rename to `low_level` to fix the `Unknown word` lint?

Reply:
Renamed